projects/igniteui-angular/src/lib/core/touch-annotations.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * Stripped-down HammerJS annotations.
+ */
+
+/**
+* @hidden
+* @internal
+*/
+export interface HammerInput {
+    preventDefault: () => {};
+    deltaX: number;
+    deltaY: number;
+    center: { x: number; y: number; };
+    pointerType: string;
+    distance: number;
+  }
+
+/**
+* @hidden
+* @internal
+*/
+  export interface HammerStatic {
+    new(element: HTMLElement | SVGElement, options?: any): HammerManager;
+
+    Pan: Recognizer;
+    Swipe: Recognizer;
+    Press: Recognizer;
+    Tap: Recognizer;
+    TouchInput: HammerInput;
+    DIRECTION_HORIZONTAL: number;
+    DIRECTION_VERTICAL: number;
+  }
+
+/**
+* @hidden
+* @internal
+*/
+  export interface Recognizer {
+    new(options?: any): Recognizer;
+    recognizeWith(otherRecognizer: Recognizer | string): Recognizer;
+  }
+
+/**
+* @hidden
+* @internal
+*/
+  export interface RecognizerStatic {
+    new(options?: any): Recognizer;
+  }
+
+/**
+* @hidden
+* @internal
+*/
+  export interface HammerInstance {
+    on(eventName: string, callback: Function): void;
+    off(eventName: string, callback: Function): void;
+  }
```

Comment:
These are entirely unused and the rest of the declarations can be reduced quite a bit as well.
These are only so our internal build passes more/less since nothing from this is exposed public.
Well, except `CarouselHammerConfig` for some reason, and even that compiles down to number in the dist declarations:
```ts
export declare class CarouselHammerConfig extends HammerGestureConfig {
    overrides: {
        pan: {
            direction: number;
        };
    };
    static ɵfac: i0.ɵɵFactoryDeclaration<CarouselHammerConfig, never>;
    static ɵprov: i0.ɵɵInjectableDeclaration<CarouselHammerConfig>;
}
```
Though that's not super relevant